config/routes.rb

```diff
@@ -1,12 +1,25 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, defaults: { format: :html },
+                         controllers: {
+                         sessions: 'users/sessions'
+                        }
```

Comment:
- [ ] Apart from returning errors when I visit the login route, I also don't think this is necessary. Devise handles login for us, you don't need to bother defining this controller here.

  ![rout](https://user-images.githubusercontent.com/36542986/187665111-39960a44-73d2-4bce-ab5a-8f9bf2a21bda.jpg)

Reply:
Can You help me in this regard plz?

Reply:
I did not understand the issue

Reply:
Hi @Fatima-hub333 when I start your app. As expected, since I'm not logged in, it redirects me to the login path. The page displays this error.